.github/workflows/close-inactive-issues.yml

```diff
@@ -0,0 +1,32 @@
+name: Close inactive issues and PRs
+on:
+  schedule:
+    - cron: "30 1 * * *"
+  workflow_dispatch:
+permissions: read-all
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9
+        with:
+          only-labels: "status: waiting-for-feedback" # only consider issues and PRs with this label
+          days-before-stale: 30 # 1 month
+          days-before-close: 60 # 2 months after being stale
+          stale-issue-label: "status: stale"
+          stale-pr-label: "status: stale"
+          stale-issue-message: >
+            If you would like us to be able to process this issue, please provide the requested information.
+            If the information is not provided within the next 2 months, we will be unable to proceed and this issue will be closed.
+          close-issue-message: >
+            Closing due to lack of requested feedback.
```

Comment:
Closing this issue due to inactivity. If you’re still interested, let us know! Our community is small, and we need contributors like you. If you provide more details or take the lead on a solution, we’ll be happy to re-open it. Let’s build this together! 🚀